_episodes/06-control-flow.md

```diff
@@ -83,6 +108,23 @@ Therefore she restarts her REPL.
 > {: .solution}
 {: .challenge}
 
+With the new `Trebuchet` defined with a complete `AbstractArray` interface,
+Melissa tries again to modify a counterweight by index:
+
+~~~
+trebuchet = Trebuchet(500, 0.25pi)
+trebuchet[1] = 2
+print(trebuchet)
+~~~
+{: .language-julia}
+~~~
+2-element Trebuchet:
+ 500.0
+   0.7853981633974483
+2
+[2.0, 0.7853981633974483]
+~~~
+{: .output}
```

Comment:
I do find these multiline input/output blocks confusing. So I'd do the following
```suggestion
~~~
trebuchet = Trebuchet(500, 0.25pi)
~~~
{: .language-julia}

~~~
2-element Trebuchet:
 500.0
   0.7853981633974483
~~~
{: .output}

~~~
trebuchet[1] = 2
~~~
{: .language-julia}
~~~
2
~~~
{: .output}

~~~
trebuchet
~~~
{: .language-julia}

~~~
2-element Trebuchet:
 2.0
 0.7853981633974483
~~~
{: .output}
```

Reply:
Yeah, that's better.

Reply:
5f4f77f